lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -78,4 +78,24 @@ describe('<DatePicker />', () => {
 
     expect(screen.queryByRole('dialog')).toBeInTheDocument();
   });
+
+  it('Does not call onChange if same date selected', async () => {
+    const onChangeMock = jest.fn();
```

Comment:
A side note on the naming convention. We have used this variation a lot in the main codebase, would it make sense to continue?
```suggestion
    const handleChange = jest.fn();
```

Reply:
Agree :)